.nx/cache/cloud/2506.06.6.hotfix1/static/login-end.html

```
@@ -0,0 +1,414 @@
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)